js/main.js

```diff
@@ -0,0 +1,114 @@
+xhr.send();
+
+function loadData(event) {
+  for (var i = 0; i < xhr.response.length; i++) {
+    var $li = document.createElement('li');
+    $li.className = 'column-third half hidden';
+    $productList.appendChild($li);
+
+    var $div = document.createElement('div');
+    $div.className = 'border-div';
+    $li.appendChild($div);
+
+    var $div1 = document.createElement('div');
+    $div1.className = 'row product';
+    $div.appendChild($div1);
+
+    var $div2 = document.createElement('div');
+    $div2.className = 'column-half';
+    $div1.appendChild($div2);
+
+    var $img = document.createElement('img');
+    $img.setAttribute('src', xhr.response[i].api_featured_image);
+    $div2.appendChild($img);
+
+    var $div3 = document.createElement('div');
+    $div3.className = 'column-half info';
+    $div1.appendChild($div3);
+
+    var $h4 = document.createElement('h4');
+    $h4.textContent = xhr.response[i].name;
+    $div3.appendChild($h4);
+
+    var $span = document.createElement('span');
+    $div3.appendChild($span);
+
+    var $i = document.createElement('i');
+    if (xhr.response[i].rating >= 0.5 && xhr.response[i].rating < 1) {
+      $i.className = 'fa-solid fa-star-half-stroke';
+    } else if (xhr.response[i].rating >= 1) {
+      $i.className = 'fa-solid fa-star';
+    } else {
+      $i.className = 'fa-regular fa-star';
+    }
+    $span.appendChild($i);
+
+    $i = document.createElement('i');
+    if (xhr.response[i].rating >= 1.5 && xhr.response[i].rating < 2) {
+      $i.className = 'fa-solid fa-star-half-stroke';
+    } else if (xhr.response[i].rating >= 2) {
+      $i.className = 'fa-solid fa-star';
+    } else {
+      $i.className = 'fa-regular fa-star';
+    }
+    $span.appendChild($i);
+
+    $i = document.createElement('i');
+    if (xhr.response[i].rating >= 2.5 && xhr.response[i].rating < 3) {
+      $i.className = 'fa-solid fa-star-half-stroke';
+    } else if (xhr.response[i].rating >= 3) {
+      $i.className = 'fa-solid fa-star';
+    } else {
+      $i.className = 'fa-regular fa-star';
+    }
+    $span.appendChild($i);
+
+    $i = document.createElement('i');
+    if (xhr.response[i].rating >= 3.5 && xhr.response[i].rating < 4) {
+      $i.className = 'fa-solid fa-star-half-stroke';
+    } else if (xhr.response[i].rating >= 4) {
+      $i.className = 'fa-solid fa-star';
+    } else {
+      $i.className = 'fa-regular fa-star';
+    }
+    $span.appendChild($i);
+
+    $i = document.createElement('i');
+    if (xhr.response[i].rating >= 4.5 && xhr.response[i].rating < 5) {
+      $i.className = 'fa-solid fa-star-half-stroke';
+    } else if (xhr.response[i].rating === 5) {
+      $i.className = 'fa-solid fa-star';
+    } else {
+      $i.className = 'fa-regular fa-star';
+    }
+    $span.appendChild($i);
+
+    var $p = document.createElement('p');
+    $p.textContent = '$' + xhr.response[i].price;
+    $div3.appendChild($p);
+  }
+}
+
+document.addEventListener('keydown', handleEnter);
```

Comment:
Instead of handling this with the `keydown` event, you should handle it with the `submit` event on the `form` element. Pressing Enter while a form control element within a form is focused will trigger the `submit` event automatically 🤓